config/kernel/linux-mvebu-current.config

```diff
@@ -2694,18 +2716,18 @@ CONFIG_MT76_USB=m
 CONFIG_MT76x02_LIB=m
 CONFIG_MT76x02_USB=m
 CONFIG_MT76x0_COMMON=m
-# CONFIG_MT76x0U is not set
+CONFIG_MT76x0U=m
 CONFIG_MT76x0E=m
 CONFIG_MT76x2_COMMON=m
 CONFIG_MT76x2E=m
 CONFIG_MT76x2U=m
-CONFIG_MT7603E=m
-CONFIG_MT7615E=m
+# CONFIG_MT7603E is not set
+# CONFIG_MT7615E is not set
```

Comment:
Most changes put something from "is not set" to module. Why do we disable these WiFi Cards? Atleast the MT7615 is tested and works. Should remain "m" imho.

Reply:
I only focused to the part below "File systems" ... other changes are random. The idea is that one person fix one section otherwise its a lot.

Reply:
Ah now I understand - just checked the Jira. 

So you want to have one kernel config for all boards in the end?

Reply:
Actually two, 32bit will remain separate. We should provide the same config at least on the same kernels. And going this way, we will see where and if we would need to make a compromise or adjust things.